AxonDeepSeg/patch_management_tools.py

```diff
@@ -67,7 +43,7 @@ def im2patches_overlap(img, overlap_value=25, scw=512):
     L_patches = []
     for e in L_pos:
         patch = img[e[0]:e[0] + scw, e[1]:e[1] + scw]
-        L_patches.append(apply_legacy_preprocess(patch))
+        L_patches.append(patch)
```

Comment:
Where has the pre-processing step been moved?

Reply:
As I mentioned today I have omitted preprocessing(histogram equalization). I will definitely see to it to see if its worth including.